### PR TITLE
View extension method for scaling view by maintaining aspecg ratio.

### PR DESCRIPTION
**Pull request by Kietyo**

### CODE REVIEW

File: korim/src/commonMain/kotlin/com/soywiz/korim/color/MaterialColors.kt [hunk 1]
```diff
@@ -0,0 +1,277 @@
+package com.soywiz.korim.color
+
+// https://material.io/design/color/the-color-system.html#tools-for-picking-colors
+object MaterialColors {
```
Comment (soywiz) [inline korim/src/commonMain/kotlin/com/soywiz/korim/color/MaterialColors.kt:4]: Isn't this from another PR?

**Review by soywiz — APPROVED** [review]
Can you remove MaterialColors first from the PR?

**Comment by Kietyo** [comment]
> 

Not sure how. Shouldn't be an issue though.

**Comment by Kietyo** [comment]
Ok seems like I fixed it somehow.

**Comment by soywiz** [comment]
Thanks!

**Pull request merged**